src/panels/lovelace/components/hui-card-options.ts

```diff
@@ -133,6 +133,10 @@ export class HuiCardOptions extends LitElement {
       paper-item {
         cursor: pointer;
       }
+
+      paper-item.delete-item {
+        color: red;
```

Comment:
```suggestion
        color: var(--google-red-500);
```

Reply:
Thanks for the feedback.

I have changed it to google-red, if this is what you prefer I would recommend changing the other instances of color: red in your project.